src/System.Net.HttpListener/src/System/Net/Windows/WebSockets/WebSocketBase.cs

```diff
@@ -1210,7 +1210,7 @@ internal void ThrowIfClosedOrAborted()
             if (State == WebSocketState.Closed || State == WebSocketState.Aborted)
             {
                 throw new WebSocketException(WebSocketError.InvalidState,
-                    SR.Format(SR.net_WebSockets_InvalidState_ClosedOrAborted, GetType().FullName, State));
+                    SR.Format(SR.net_WebSockets_InvalidState_ClosedOrAborted, GetType().ToString(), State));
```

Comment:
Why are these changing from FullName to ToString()?  Doesn't desktop use FullName?
https://referencesource.microsoft.com/#System/net/System/Net/WebSockets/WebSocketBase.cs,1264